packages/nocodb/src/db/BaseModelSqlv2.ts

```
@@ -2158,6 +2174,23 @@ class BaseModelSqlv2 implements IBaseModelSqlV2 {
+                columns,
+              );
+              selectors.push(
+                this.dbDriver.raw('(??)::jsonb as ??', [
+                  columnName,
+                  getAs(column),
+                ]),
+              );
+              groupBySelectors.push(getAs(column));
+              break;
+            }
+          }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)